qemu/tests/cfg/qemu_img_convert_from_vdi.cfg

```diff
@@ -0,0 +1,19 @@
+- qemu_img_convert_from_vdi:
+    only RHEL
+    only i440fx
+    no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8
+    required_qemu = [6.1.0-7, )
+    virt_test_type = qemu
+    type = qemu_img_convert_from_vdi
+    start_vm = no
+    kill_vm = yes
+    vdi_image_address = "http://fileshare.englab.nay.redhat.com/pub/section2/kvm/timao/vdi_image/rhel850.vdi"
```

Comment:
never attach 'private info' in upstream

Reply:
Thanks, have added the info to the test loop.